functions/disable-functions.env

```diff
@@ -12,4 +12,5 @@ AzureWebJobs.prepared-check-sync.Disabled=true
 AzureWebJobs.pupil-login.Disabled=true
 AzureWebJobs.pupil-prefs.Disabled=true
 AzureWebJobs.report-psychometrician.Disabled=true
-AzureWebJobs.report-psychometrician-create-download.Disabled=true
\ No newline at end of file
+AzureWebJobs.report-psychometrician-create-download.Disabled=true
+AzureWebJobs.test-pupil-connection.Disabled=true
```

Comment:
this is for disabling functions.  we are not including a function to process the queue messages. they will just expire after 7 days.  i think you should remove this?

Reply:
@GuyHarwood yes absolutely that is definitely redundant. removed.